src/install.rs

```diff
@@ -233,6 +236,53 @@ fn write_platform(mountpoint: &Path, platform: &str) -> Result<()> {
     Ok(())
 }
 
+/// Copy networking config if asked to do so
+fn copy_network_config(mountpoint: &Path, net_config_src: &str) -> Result<()> {
+    eprintln!("Copying networking configuration from {}", net_config_src);
+
+    // get the path to the destination directory
+    let mut net_config_dest = mountpoint.to_path_buf();
+    net_config_dest.push("coreos-installer-network");
+
+    // make the directory if it doesn't exist
+    create_dir_all(&net_config_dest)
+        .chain_err(|| {
+            format!(
+                "creating destination networking config directory {}",
+                net_config_dest.display()
+            )
+        })?;
+
+    // if the directory has files in it then clear them out
+    for entry in read_dir(&net_config_dest)
+        .chain_err(|| format!("reading directory {}", net_config_dest.display()))?
+    {
+        let entry = entry
+            .chain_err(|| format!("reading directory {}", net_config_dest.display()))?;
+        if entry.path().is_file() {
+            eprintln!("Removing file {}", entry.path().display());
+            remove_file(entry.path()).
+                chain_err(|| format!("removing {}", entry.path().display()))?;
+        }
+    }
```

Comment:
I'll delete this code as this should never happen because we just wrote out the fresh disk image to the block device.